.github/workflows/pr.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     if: contains(github.event.pull_request.labels.*.name, 'safe-to-test')
     env:
       MAPI_TOKEN: ${{ secrets.MAPI_TOKEN }}
```

Comment:
Since tokens don't migrate, I think we can remove all references to `MAPI_TOKEN` and `mapi-token`.  Same thing with `MAPI_URL` and `mapi-url`.